<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwf2options</class>
 <widget class="QDialog" name="pwf2options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pwf2main.qrc">
    <normaloff>:/img/pwf2tools.png</normaloff>:/img/pwf2tools.png</iconset>
  </property>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="settingsTabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="PrivacyTab">
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <attribute name="title">
     <string>Privacy</string>
    </attribute>
    <widget class="QCheckBox" name="enableRPCbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable DiscordRPC</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="userExpBox">
    <attribute name="title">
     <string>User Experience</string>
    </attribute>
    <widget class="QCheckBox" name="enableToolclose">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Close window on tool use</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pwf2main.qrc"/>
 </resources>
 <connections/>
</ui>
